diff --git a/CMakeLists.txt b/CMakeLists.txt
index 5ce5aa95..41b933b2 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -1000,6 +1000,7 @@ file(GLOB KERNEL_SRC
 	${PROJECT_SOURCE_DIR}/kernel/generic/kernel_dgetrf_pivot_lib4.c
 	${PROJECT_SOURCE_DIR}/kernel/generic/kernel_dgeqrf_4_lib4.c
 	${PROJECT_SOURCE_DIR}/kernel/generic/kernel_dpack_lib4.c
+ 	${PROJECT_SOURCE_DIR}/kernel/generic/kernel_dpack_buffer_lib4.c
 	${PROJECT_SOURCE_DIR}/kernel/generic/kernel_ddot_lib.c
 	${PROJECT_SOURCE_DIR}/kernel/generic/kernel_daxpy_lib.c
 	${PROJECT_SOURCE_DIR}/kernel/generic/kernel_dgetr_lib.c
