From 8b0246a82c2b7ce9574ee667fff54ad931b5ebdd Mon Sep 17 00:00:00 2001
From: Silvio Traversaro <silvio@traversaro.it>
Date: Tue, 3 Sep 2024 15:47:48 +0200
Subject: [PATCH] Do not build tests that require blas

---
 tests/CMakeLists.txt | 12 ++++++------
 1 file changed, 6 insertions(+), 6 deletions(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 5ce5aa95..2d12d24a 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -590,6 +590,7 @@ file(GLOB BLASFEO_HP_PM_SRC
        ${PROJECT_SOURCE_DIR}/blasfeo_hp_pm/s_blas3_lib8.c
        ${PROJECT_SOURCE_DIR}/blasfeo_hp_pm/s_blas3_diag_lib8.c
        ${PROJECT_SOURCE_DIR}/blasfeo_hp_pm/s_lapack_lib8.c
+       ${PROJECT_SOURCE_DIR}/kernel/generic/kernel_dpack_buffer_lib4.c
        )

 file(GLOB AUX_HP_PM_SRC
@@ -1000,6 +1001,7 @@ file(GLOB KERNEL_SRC
        ${PROJECT_SOURCE_DIR}/kernel/generic/kernel_dgetrf_pivot_lib4.c
        ${PROJECT_SOURCE_DIR}/kernel/generic/kernel_dgeqrf_4_lib4.c
        ${PROJECT_SOURCE_DIR}/kernel/generic/kernel_dpack_lib4.c
+       ${PROJECT_SOURCE_DIR}/kernel/generic/kernel_dpack_buffer_lib4.c
        ${PROJECT_SOURCE_DIR}/kernel/generic/kernel_ddot_lib.c
        ${PROJECT_SOURCE_DIR}/kernel/generic/kernel_daxpy_lib.c
        ${PROJECT_SOURCE_DIR}/kernel/generic/kernel_dgetr_lib.c

diff --git a/tests/CMakeLists.txt b/tests/CMakeLists.txt
index 19c351d7..67403958 100644
--- a/tests/CMakeLists.txt
+++ b/tests/CMakeLists.txt
@@ -36,17 +36,17 @@
 
 add_executable(test_d_blasfeo_api test_d_blasfeo_api.c)
 add_executable(test_s_blasfeo_api test_s_blasfeo_api.c)
-add_executable(test_d_blas_api test_d_blas_api.c)
-add_executable(test_s_blas_api test_s_blas_api.c)
+#add_executable(test_d_blas_api test_d_blas_api.c)
+#add_executable(test_s_blas_api test_s_blas_api.c)
 
 if(CMAKE_C_COMPILER_ID MATCHES MSVC) # no explicit math library
 	target_link_libraries(test_d_blasfeo_api blasfeo)
 	target_link_libraries(test_s_blasfeo_api blasfeo)
-	target_link_libraries(test_d_blas_api blasfeo)
-	target_link_libraries(test_s_blas_api blasfeo)
+	#target_link_libraries(test_d_blas_api blasfeo)
+	#target_link_libraries(test_s_blas_api blasfeo)
 else() # add explicit math library
 	target_link_libraries(test_d_blasfeo_api blasfeo m)
 	target_link_libraries(test_s_blasfeo_api blasfeo m)
-	target_link_libraries(test_d_blas_api blasfeo m)
-	target_link_libraries(test_s_blas_api blasfeo m)
+	#target_link_libraries(test_d_blas_api blasfeo m)
+	#target_link_libraries(test_s_blas_api blasfeo m)
 endif()
