diff --git a/CMakeLists.txt b/CMakeLists.txt
index 5ce5aa95..2d12d24a 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -590,6 +590,7 @@ file(GLOB BLASFEO_HP_PM_SRC
        ${PROJECT_SOURCE_DIR}/blasfeo_hp_pm/s_blas3_lib8.c
        ${PROJECT_SOURCE_DIR}/blasfeo_hp_pm/s_blas3_diag_lib8.c
        ${PROJECT_SOURCE_DIR}/blasfeo_hp_pm/s_lapack_lib8.c
+       ${PROJECT_SOURCE_DIR}/kernel/generic/kernel_dpack_buffer_lib4.c
        )

 file(GLOB AUX_HP_PM_SRC
