diff --git a/CMakeLists.txt b/CMakeLists.txt
index 5a42cd44..1edca8c0 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -887,10 +887,10 @@ file(GLOB KERNEL_SRC
 	${PROJECT_SOURCE_DIR}/kernel/armv8a/kernel_dgemm_8x4_lib4.S
 	${PROJECT_SOURCE_DIR}/kernel/armv8a/kernel_dgemm_4x4_lib4.S
 	${PROJECT_SOURCE_DIR}/kernel/armv8a/kernel_dpack_lib4.S
-	${PROJECT_SOURCE_DIR}/kernel/armv8a/kernel_dgetrf_pivot_lib4.S
+	${PROJECT_SOURCE_DIR}/kernel/armv8a/kernel_dgetrf_pivot_lib4.c
 	${PROJECT_SOURCE_DIR}/kernel/armv8a/kernel_dgemv_4_lib4.S
 	${PROJECT_SOURCE_DIR}/kernel/armv8a/kernel_dgetr_lib.S
-	${PROJECT_SOURCE_DIR}/kernel/armv8a/kernel_dger_lib4.c
+	${PROJECT_SOURCE_DIR}/kernel/armv8a/kernel_dger_lib4.S
 	${PROJECT_SOURCE_DIR}/kernel/generic/kernel_dgemm_4x4_lib4.c
 	${PROJECT_SOURCE_DIR}/kernel/generic/kernel_dgemm_diag_lib4.c
 	${PROJECT_SOURCE_DIR}/kernel/generic/kernel_dgemv_4_lib4.c
@@ -940,10 +940,10 @@ file(GLOB KERNEL_SRC
 	${PROJECT_SOURCE_DIR}/kernel/armv8a/kernel_dgemm_8x4_lib4.S
 	${PROJECT_SOURCE_DIR}/kernel/armv8a/kernel_dgemm_4x4_lib4.S
 	${PROJECT_SOURCE_DIR}/kernel/armv8a/kernel_dpack_lib4.S
-	${PROJECT_SOURCE_DIR}/kernel/armv8a/kernel_dgetrf_pivot_lib4.S
+	${PROJECT_SOURCE_DIR}/kernel/armv8a/kernel_dgetrf_pivot_lib4.c
 	${PROJECT_SOURCE_DIR}/kernel/armv8a/kernel_dgemv_4_lib4.S
 	${PROJECT_SOURCE_DIR}/kernel/armv8a/kernel_dgetr_lib.S
-	${PROJECT_SOURCE_DIR}/kernel/armv8a/kernel_dger_lib4.c
+	${PROJECT_SOURCE_DIR}/kernel/armv8a/kernel_dger_lib4.S
 	${PROJECT_SOURCE_DIR}/kernel/generic/kernel_dgemm_4x4_lib4.c
 	${PROJECT_SOURCE_DIR}/kernel/generic/kernel_dgemm_diag_lib4.c
 	${PROJECT_SOURCE_DIR}/kernel/generic/kernel_dgemv_4_lib4.c
